languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/german_prefix.txt

```diff
@@ -0,0 +1,927 @@
+sach
```

Comment:
As jwordsplitter is a dependency, this should be available in class path, so we can probably just load it from there without duplicating the file.

Reply:
Could you share a hint on how to do that? I noticed that the `germanPrefixes.txt` has been excluded in the jwordsplitter's `pom.xml`, see [here](https://github.com/danielnaber/jwordsplitter/blob/77d8a87f0ed4f07e0d58e70943b9a39b84bf047d/pom.xml#L125).

Reply:
> I noticed that the germanPrefixes.txt has been excluded in the jwordsplitter's pom.xml, see [here](https://github.com/danielnaber/jwordsplitter/blob/77d8a87f0ed4f07e0d58e70943b9a39b84bf047d/pom.xml#L125).

Oh, I didn't remember that. In that case, my idea won't work. Could you add a comment (`# ...`) at the top of the file that explains where it comes from? Please let me know if the PR is ready.

Reply:
I added a comment at the the top of germanprefixes.txt! PR is ready for review! Thanks 💪